examples/sub-module/test-modules-size.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
```

Comment:
Should have at least `set -e` so it exists the entire script if there is a error